### PR TITLE
Display account transactions

### PR DESCRIPTION
**Pull request by MaxStalker**
Implements #9

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -5,3 +5,8 @@ body {
   margin: 0;
   min-height: 100%;
 }
+
+h1{
```
Comment (longfin) [inline src/index.css]: For consistency 😄 

```suggestion
h1 {
```

File: schema.graphql [hunk 2]
```diff
@@ -0,0 +1,81 @@
+# This file was generated based on ".graphqlconfig". Do not edit manually.
```
Comment (longfin) [inline schema.graphql]: This file seems the same file as `src/schema.graphql`, is it necessary?

**Comment by limebell** [comment]
Thanks for your contribution!

File: src/pages/block.tsx [hunk 3]
```diff
@@ -98,7 +115,10 @@ const TxList: React.FC<TxListProps> = ({ txs }) => {
       isSorted: false,
       isSortedDescending: false,
       data: 'string',
-      isPadded: true
+      isPadded: true,
+      onRender: tx => (
+        <Link href={`/account/?${tx.signer}`}>{tx.id}</Link>
```
Comment (earlbread) [inline src/pages/block.tsx]:
```suggestion
        <Link href={`/account/?${tx.signer}`}>{tx.signer}</Link>
```

File: schema.graphql (hunk 2 above)
Comment (MaxStalker) [reply schema.graphql]: Indeed. That was something brought in by IDE. I've removed it now.

**Pull request merged**